CMB2/CMB2
# Fix issue where image can be attached to wrong group after removing previous group.

## Description
Fixes #1469

## Testing procedure
See steps in issue: https://github.com/CMB2/CMB2/issues/1469#issue-1422487256
1. Make sure you can reproduce the bug in the issue.
2. Pull down this PR
3. Test again
4. See that this PR addresses the issue and the file is correctly associated to the group.

## Types of changes
- **Bug fix (non-breaking change which fixes an issue)**
